### ISSUE TITLE
Missing XML declaration from `omexmlClass.OMEXML().to_xml`

### ISSUE THREAD

**Issue by manzt** [apeer-micro/apeer-ometiff-library#12]
Thanks for creating this tool!

I have been trying to convert numpy arrays into OME-TIFFs and this package has been very useful. However, the XML string generated from `str(omexmlClass.OMEXML())` or `omexmlClass.OMEXML().to_xml()` is missing the [XML declaration](https://en.wikipedia.org/wiki/XML): 

```
<?xml version="1.0" encoding="UTF-8"?>
```

which https://github.com/cgohlke/tifffile uses to determine if the tiff is an OME-TIFF and parse the metadata. I have found this to be a decent workaround:

```python
metadata = omexmlClass.OMEXML()
omexml_string = f'<?xml version="1.0" encoding="UTF-8"?>\n{metadata}'
```

But it took digging deeper into the libraries to find the issue.

**Comment by DaniGodin** [comment]
Hi, 
Thx for the feedback, this is now fixed in the 1.7.0 update!

**Comment by DaniGodin** [comment]
@manzt  Could you please close the issue :)
